### PR TITLE
Try fix API docs

### PR DESCRIPTION
**Pull request by jfversluis**
Little fix for our API docs

Related to #3960

### CODE REVIEW

**Comment by Eilon** [comment]
Interesting. Can you share what was broken and how this fixes it? Back when I made _most_ of the API docs work, I didn't notice any issues, but with thousands of docs, I certainly didn't check everything 😄

**Comment by jfversluis** [comment]
@Eilon the essence is captured here: https://github.com/mono/api-doc-tools/issues/650

Let me know if it's not clear, I can explain a bit more offline

**Comment by Eilon** [comment]
Ah very interesting, this does make sense!

File: .nuspec/maui.aotprofile.txt [hunk 1]
```diff
@@ -4125,7 +4125,7 @@ Methods:
 	void Microsoft.Maui.Controls.CheckBox/<>c:.cctor ()
 	void Microsoft.Maui.Controls.CheckBox/<>c:.ctor ()
 	void Microsoft.Maui.Controls.CollectionView:.ctor ()
-	void Microsoft.Maui.Controls.ColorElement:.cctor ()
+	void Microsoft.Maui.Graphics.ColorElement:.cctor ()
```
Comment (jfversluis) [inline .nuspec/maui.aotprofile.txt:4128]: Not sure about this one, does this look right? 😅

**Comment by Eilon** [comment]
BTW please check in your local VS whether these changes work. That's how I found out a long time ago that much of the doc comments we had were being entirely ignored. So with whatever changes you have here, just hover over various classes, methods, events, etc. and see that all the docs that exist are actually showing up.

**Comment by jfversluis** [comment]
@Eilon definitely doing that! Although the primary goal of this is to get them up and running in the online API browser. Once we've done that, I am going to work to improve these inline docs as a whole for both IntelliSense as well as the online API docs browser

**Review by rmarinho — APPROVED** [review]

**Pull request merged**